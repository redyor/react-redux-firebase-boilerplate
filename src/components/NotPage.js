import React from 'react';
import { Link } from 'react-router-dom';
const NotPage = () => (
  <div>
    404 Not found! <Link to="/">Home</Link>
  </div>
);
export default NotPage;
