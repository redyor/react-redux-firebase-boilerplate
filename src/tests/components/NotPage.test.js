import React from 'react';
import { shallow } from 'enzyme';
import NotPage from '../../components/NotPage';

it('Should Render not found page', () => {
  const wrapper = shallow(<NotPage />);
  expect(wrapper).toMatchSnapshot();
});
