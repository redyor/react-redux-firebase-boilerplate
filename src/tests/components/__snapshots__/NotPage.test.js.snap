// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render not found page 1`] = `
<div>
  404 Not found! 
  <Link
    to="/"
  >
    Home
  </Link>
</div>
`;
